aries-site/src/pages/templates/app-layouts.mdx

```diff
@@ -22,39 +35,27 @@ Page Container.
   <AppAnatomy />
 </Example>
 
-1. **App Container**  
-The container serves as the outer shell of a page UI. This is the 
-parent component containing all HTML elements within a grid. It's 
-responsibility is to organize each of the page components, how they 
-relate to each other, the spacing between each, and how they respond 
-at various screen widths.
+1. **App Container**
+- The container serves as the outer shell of a page UI.
+- This is the parent component containing all HTML elements within a grid.
+- It's responsibility is to organize each of the page components, how they 
+relate to eachother, the spacing between each, and how they respond at 
+various screen widths.
 
 2. **Global Header & Footer**  
-The Global Header, also known as the HPE Common Header and Footer 
-Web Service, provides a tunable, yet consistent header and footer 
-to be used across all HPE applications. For more information, see 
-[Global Header](/templates/global-header).
+- The Global Header, also known as the [HPE Common Header and Footer 
+Web Service](/templates/global-header), provides a tunable, yet consistent 
+header and footer to be used across all HPE applications.
+- No matter the screen size, the Global Header & Footer will always be full 
+width, occupying as much space as available.
 
 3. **Page Container**  
-The Page Container contains the Page Content and the Page Header. 
-It is responsible for the layout for the main content of a page. 
+- The Page Container contains the Page Content and the Page Header.
+- It is responsible for the layout for the main content of a page. 
+- Page container width may be [Wide](#wide-default), [Narrow](#narrow), of [Full](#full).
 
-<Example
-  caption='Diagram of a Page Container which contains a Page Header 
-    and the Page Content.'
-  pad="none"
-  previewWidth="min-content"
->
-  <PageAnatomy />
-</Example>
-
-Jump to:
-- [Container Width](#page-container-widths)
-- [Grid](#grid)
-- [Breakpoints](#breakpoints)
-
-Coming soon:
-- Page Content layout templates for Dashboards, Wizards, Lists and Tables.
+🏗️ **Coming soon:**
```

Comment:
I found this icon a little bit different to see / interpret given the size -- could we do something like construction cone or construction sign? something with a little less detail

Reply:
@bahriaditi